client/src/components/EventTypeDialog.js

```diff
@@ -78,7 +78,42 @@ export default function EventTypeDialog() {
 
     const handleColorChange = (color) => {
         setColor('#'+color.hex)
-    }
+    };
+    
+    const handleConfirm = () => {
+        if (title===''||duration===''||url===''||color==='') return alert('Missing Field(s)') //might need to change after discuss
```

Comment:
I will be making a task later to change alerts into a proper alert system app-wide so that we can show messages to the users in a better UI